src/lib.rs

```diff
@@ -315,6 +316,17 @@ impl VadSession {
                                 - self.silent_samples)
                                 / (self.config.sample_rate / 1000);
 
+                            // Since speech_end_ms does not include silent_samples, the post
+                            // padding should be added to speech_end_ms.
+                            let speech_end_with_pad_ms =
```

Comment:
You don't seem to use this anywhere? But it should be reflected in the emitted `SpeechEnd` event and the cached active speech.